electrum/util.py

```diff
@@ -767,7 +767,7 @@ def format_satoshis_plain(
 
 # DECIMAL_POINT = locale.localeconv()['decimal_point']  # type: str
 DECIMAL_POINT = "."
-THOUSANDS_SEP = " "
+THOUSANDS_SEP = ","
```

Comment:
Thank you for making this change. This has been bothering me for awhile.
Can this get merged ASAP?